### PR TITLE
Refine the plot categories and define a default set of plots for amwg

### PR DESCRIPTION
**Pull request by cecilehannay**
I updated the plot categories so we don't have so many plots in a each category.
I also defined in config_amwg_default_plots.yaml the default set of variable we want to plot for amwg development.

### CODE REVIEW

**Review by nusbaume — CHANGES_REQUESTED** [review]
Looks good, but I did have some suggestions for the new config file that would be good to get your thoughts on.

File: config_amwg_default_plots.yaml [hunk 1]
```diff
@@ -0,0 +1,403 @@
+#==============================
+#config_cam_baseline_example.yaml
```
[the remark this answers is not in the document]
Comment (cecilehannay) [reply config_amwg_default_plots.yaml:2]: I agree. I changed that.

File: config_amwg_default_plots.yaml [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+    #Overwrite time series files, if found?
+    #If set to false, then time series creation will be skipped if files are found:
+    cam_overwrite_ts: false
+
+    #Location where time series files are (or will be) stored:
+    cam_ts_loc: /glade/scratch/${user}/ADF/${diag_cam_climo.cam_case_name}/ts
+
+    #----------------------
+
+    #You can alternatively provide a list of cases, which will make the ADF
```
[the remark this answers is not in the document]
Comment (cecilehannay) [reply config_amwg_default_plots.yaml:191]: Done

File: config_amwg_default_plots.yaml [hunk 3]
```diff
@@ -0,0 +1,403 @@
...
+#List of plotting scripts being used.
+#These scripts must be located in "scripts/plotting":
+plotting_scripts:
+    - global_latlon_map
+    - global_latlon_vect_map
+    - zonal_mean
+    - meridional_mean
+    - polar_map
+    - cam_taylor_diagram
+#    - qbo
```
[the remark this answers is not in the document]
Comment (cecilehannay) [reply config_amwg_default_plots.yaml:354]: I am not sure I agree with that one. This is the default set of plots we are doing to evaluate cam7 runs for the AMWG.

[the remark this answers is not in the document]
Comment (nusbaume) [reply config_amwg_default_plots.yaml:354]: Sorry, "this entry" means explicitly the `qbo` plot.  If the QBO is supposed to be included as part of the CAM7 evaluation runs then should it be un-commented here?

[the remark this answers is not in the document]
Comment (cecilehannay) [reply config_amwg_default_plots.yaml]: Oh, it makes sense. I had deleted it already in the new file. So I think we are good to go. 
Thanks a bunch.

**Review by nusbaume — APPROVED** [review]
Everything looks good now.  Thanks @cecilehannay!

**Pull request merged**